### DISCUSSION TITLE
打造电商平台，如何获取公共组件和通用信息，比如商品信息以及banner信息，公告推送消息推送是否支持

### DISCUSSION THREAD

**Discussion by Toek** [halo-dev/halo#4346] [Q&A]
### Your current Halo version

2.7.0

### Describe this feature

如果想打造一个电商平台，电商平台常用的业务场景公共组件，公共接口是否支持？具体是哪些接口支持的？

### Additional information

比如，获取电商通用信息如何获取，通过哪些接口：例如1.商品信息 2.banner信息。

再比如，实时消息推送功能是否支持：比如常用的类似公告消息业务场景，框架是否支持消息推送，是否有排上日程。

**Comment by JohnNiang** [comment]
我所了解的消息通知机制已经在 https://github.com/halo-dev/halo/issues/4045 提到，欢迎持续跟进。

**Comment by Toek** [comment]
> 我所了解的消息通知机制已经在 #4045 提到，欢迎持续跟进。

那么第1&2两个细分小问题麻烦大佬也解答下呢（1.商品信息 & 2.banner信息）。

**Comment by ruibaby** [comment]
> 比如，获取电商通用信息如何获取，通过哪些接口：例如1.商品信息 2.banner信息。

不太清楚你具体需要实现成什么样子，我们在 https://halo.run/store/apps 的实践是这样的：

1. 主题配合插件来实现应用市场的整个架构。
2. 应用通过插件提供的 Console 页面进行管理。
3. Banner 这种信息的设置实现方式有很多，可以通过主题设置，也可以通过插件设置。
